backend/service/status/status_test.go

```diff
@@ -38,6 +39,16 @@ var (
 	httpAddr   = testutils.CreateURLs("http", "")
 )
 
+var jsonContentHeaders = map[string]string{
+	header.Accept:      header.ApplicationJSON,
+	header.ContentType: header.ApplicationJSON,
+}
+
+var textContentHeaders = map[string]string{
+	header.Accept:      header.TextPlain,
+	header.ContentType: header.ApplicationJSON,
```

Comment:
Should not the contentype header also be header.TextPlain?

Reply:
`Content-Type` what is sent. 
`Accept` was it wants back